slices/iproute2.yaml

```diff
@@ -0,0 +1,75 @@
+package: iproute2
+
+slices:
+  # iproute2 is a utility package which typically could be split into
+  # utility-specific slices. The `bins` slice however brings in all the
+  # utilities for cases where iproute2 is used as a dependency for another
+  # package (eg. nginx), where we can't easily know which utilities are
+  # actually going to be needed.
+  bins:
+    # Leaving the debconf dependency out as we don't currently need pkg mgmt.
+    # Also, debconf is used by the pkg's maintainer scripts, which are handled
+    # differently in Chisel. 
+    essential:
+      - iproute2_config
+      - iproute2_libs
+      - libcap2-bin_bins
+    contents:
+      /bin/ip:
+      /sbin/ip:
+      /bin/ss:
+      /sbin/bridge:
+      /sbin/dcb:
+      /sbin/devlink:
+      /sbin/rtacct:
+      /sbin/rtmon:
+      # "tc" is an obvious candidate for getting its own "traffic-control"
+      # slice in the future, if needed.
+      /sbin/tc:
+      /sbin/tipc:
+      /sbin/vdpa:
+      /usr/bin/lnstat:
+      /usr/bin/ctstat:
+      /usr/bin/rtstat:
+      /usr/bin/nstat:
+      /usr/bin/rdma:
+      /usr/bin/routel:
+      /usr/sbin/arpd:
+      /usr/sbin/genl:
+
+  config:
+    # Corresponds to the pkg's conffiles
+    contents:
+      /etc/iproute2/bpf_pinning:
+      /etc/iproute2/ematch_map:
+      /etc/iproute2/group:
+      /etc/iproute2/nl_protos:
+      /etc/iproute2/rt_dsfield:
+      /etc/iproute2/rt_protos:
+      /etc/iproute2/rt_protos.d/:
+      /etc/iproute2/rt_realms:
+      /etc/iproute2/rt_scopes:
+      /etc/iproute2/rt_tables:
+      /etc/iproute2/rt_tables.d/:
+
+  libs:
+    essential:
+      - libbpf1_libs
+      - libbsd0_libs
+      - libc6_libs
+      - libcap2_libs
+      - libdb5.3_libs
+      - libelf1_libs
+      - libmnl0_libs
+      - libselinux1_libs
+      - libgssapi-krb5-2_libs
```

Comment:
Should this have been `libtirpc3_libs` instead?

Reply:
y just noticed my typo. nice catch ;) thanks. fixed in https://github.com/canonical/chisel-releases/pull/59/commits/71e5c3d671999b5610be5defeff5c4dc7f891835